specs/fulu/p2p-interface.md

```diff
@@ -266,6 +267,27 @@ gossip. In particular, clients MUST:
 
 #### Messages
 
+##### Status v2
+
+**Protocol ID:** `/eth2/beacon_chain/req/status/2/`
+
+Request and Response remain unchanged unless explicitly noted here.
+
+```
+(
+  fork_digest: ForkDigest
+  finalized_root: Root
+  finalized_epoch: Epoch
+  head_root: Root
+  head_slot: Slot
+  lowest_slot: Slot # [New in Fulu:EIP7594]
+)
+```
+
+The added fields are, as seen by the client at the time of sending the message:
```

Comment:
What's the purpose of "as seen by the client at the time of sending the message"? Maybe simplify:
```suggestion
The following fields are added:
```